core/string.c

```diff
@@ -117,28 +117,32 @@ d_r_strrchr(const char *str, int c)
 /* Private strncpy.  Standard caveat about not copying trailing null byte on
  * truncation applies.
  */
+#define D_R_STRNCPY_BODY                      \
```

Comment:
style: use parens `()` for anything non-trivial

Reply:
Done.